src/main/java/Account.java

```diff
@@ -16,19 +15,11 @@ public Account(String owner, double quantityAdded) {
     public String getOwner() {
         return owner;
     }
-
-    public void setOwner(String owner) {
-        this.owner = owner;
-    }
-
+    
     public double getQuantity() {
         return quantity;
     }
 
-    public void setQuantity(double quantity) {
```

Comment:
Why would you remove this method if it's being used in withdrawQuantity method?

Reply:
because it is a public method. You are allowing the clients of this class to set any value here.

Reply:
![image](https://user-images.githubusercontent.com/45263637/146616822-dd2435bc-5572-4e2d-aac9-6808d40bef30.png)